.ci/static-checks.sh

```diff
@@ -891,6 +892,43 @@ static_check_xml()
 	done
 }
 
+static_check_shell()
+{
+	local all_scripts
+	local scripts
+
+	all_scripts=$(find . \( -name "*.sh" -o -name "*.bash" \) | grep -v "/vendor/" | sort || true)
+
+	if [ "$specific_branch" = "true" ]
+	then
+		info "Checking all scripts in $branch branch"
+		scripts="$all_scripts"
+	else
+		info "Checking local branch for changed scripts only"
+
+		local scripts_status
+		scripts_status=$(get_pr_changed_file_details || true)
+		scripts_status=$(echo "$scripts_status" | grep -E "\.(sh|bash)$" || true)
+
+		scripts=$(echo "$scripts_status" | awk '{print $NF}')
+	fi
+
+	[ -z "$scripts" ] && info "No scripts to check" && return 0
```

Comment:
you could move the 'no files' check next to the find - either up or down - right now they are separated by the branch check, and it feels like the two things (find files, check we have some and the branch checks) are two separate things?

Reply:
Well that check needs to be there since if you look closely that branch check determines what the value of `$scripts` is.